iceoryx_posh/etc/iceoryx/roudi_config_example.toml

```diff
@@ -1,48 +1,33 @@
+# Adapt this config to your needs and rename it to e.g. roudi_config.toml
```

Comment:
not "to e.g. roudi_config.toml" but just "to roudi_config.toml so that RouDi finds it at startup"

Reply:
True, will fix in follow up pull request.